src/utils/breadcrumbItems.ts

```diff
@@ -132,8 +166,9 @@ export const addUserBreadcrumb: BreadcrumbItem = {
 
 // HELPER FUNCTIONS
 export const cityFromBreadcrumb: (

```

Comment:
This function is kind of overtyped (it was already this way before this PR), let's simplify it to something like this:
```ts
export const cityFromBreadcrumb = async (type: string, from?: string[]) => {
```